### PR TITLE
Lesson 7

### PR DESCRIPTION
**Pull request by Irina-pi**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,49 +1,60 @@
...
-  return array;
-};
 
-console.log(zero(arr167));
\ No newline at end of file
+  return true;
+}
+
+console.log(deepEqual(obj1, obj2));
+console.log(deepEqual(obj1, obj3));
```
Comment (5tronciy) [inline main.js]: deepEqual({a: 'a', b: {a: 'a' }}, {a: 'a', b: {b: 'b' }}) returns true.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
We agreed that you would watch what you push to the remote repo. And I am sure you know how to delete node_modules. Do it before improving deepEqual().

File: main.js (hunk 1 above)
Comment (Irina-pi) [reply main.js]: I made function deepEqual correctly

**Review by 5tronciy — APPROVED** [review]
Works well. Approve.

**Pull request merged**